GuessTheMovie/ImageStorage.h

```diff
@@ -29,21 +33,24 @@ struct ImageStorage
 		{
 			create_folder(film_folder_path);
 		}
-		
+
 		if (is_folder_empty(film_folder_path))
 		{
 			create_missing_images(film_name, film_folder_path);
 		}
-		
+
 		return get_folder_contents(film_folder_path);
 	}
 
 	// TODO: implement this function
-	void create_folder(const std::string& path) 
+	void create_folder(const std::string& path)
 	{
-        std::cout << "Creating folder at " << path << std::endl;
-
-        std::cout << "Finished creating folder at " << path << std::endl;
+		std::cout << "Creating folder at " << path << std::endl;
+		// std::ofstream fout(path)
+		// I create in GuessTheMovie file new file ImageStorage
+		std::ofstream fout("ImageStorage\\film_folder.txt");
```

Comment:
Firstly, this won't create the "ImageStorage" folder if it doesn't exist.
Secondly, the create_folder function should create a folder, not a file!